docs/Adding-Software-to-Homebrew.md

```diff
@@ -2,7 +2,7 @@
 
 Is your favorite software missing from Homebrew? Then you're the perfect person to resolve this problem.
 
-If you want to add software that is either closed source or a GUI-only program, you will want to follow the guide for [Casks](#casks). Otherwise follow the guide for [Formulae](#formulae). See also: [Homebrew Terminology](Formula-Cookbook.md#homebrew-terminology)
+If you want to add software that is either closed source or a GUI-only program, you will want to follow the guide for [Casks](#casks). Otherwise follow the guide for [Formulae](#formulae). (see also: [Homebrew Terminology](Formula-Cookbook.md#homebrew-terminology))
```

Comment:
```suggestion
If you want to add software that is either closed source or a GUI-only program, you will want to follow the guide for [Casks](#casks). Otherwise follow the guide for [Formulae](#formulae) (see also: [Homebrew Terminology](Formula-Cookbook.md#homebrew-terminology))
```